handlers.go

```diff
@@ -2263,6 +2263,11 @@ func (s *Server) handleGetViewer(c echo.Context, u *User) error {
 		return err
 	}
 
+	var at AuthToken
+	if err := s.DB.Find(&at, "\"user\" = ? AND token = ?", u.ID, c.Param("key")).Error; err != nil {
```

Comment:
this wont work AFAIK. we will have to augment checkTokenAuth to store the auth token for the user it returns in that user object. Putting it in an unexported field seems to make sense to me. Then we can just get `u.authToken.Expiry` here